cms/djangoapps/appsembler/urls.py

```diff
@@ -0,0 +1,17 @@
+"""Perform Studio local login/logout
+
+The purpose of this is to address the issue that Ironwood introduced login
+redirect to the LMS, which breaks in multisite custom domain environments
+
+We have this code in the Appsembler CMS app to help isolate custom code
+"""
+from django.conf import settings
+from django.urls import path
+from django.contrib.auth.views import LogoutView
+from .views import LoginView
+
+urlpatterns = [
+    path('login/', LoginView.as_view(), name='login'),
```

Comment:
It seems that a couple of tests are failing due to the change login url. I think we can overcome this change by reusing the name:

While this is a hack, it keeps it in our code rather than upstream code.

```suggestion
    path('login/', LoginView.as_view(), name='login_redirect_to_lms'),   # Tahoe: This reuses the name from the redirect JWT login view to avoid failing upstream tests.
```

Reply:
@OmarIthawi Based on our conversation, we're going to make the Studio local login a configuration conditional setting. To note, I recommend against having a 'login_redirect_to_lms' name for a local login as it violates the principle of least surprise.

Reply:
That's right, let's see if feature flags help.